pkg/materialize/cluster.go

```diff
@@ -167,7 +167,8 @@ func (b *ClusterBuilder) GenerateClusterOptions() string {
 		p = append(p, i)
 	}
 
-	if b.disk {
+	// Only add DISK to the quiery builder if it's enabled AND size doesn't end in either "cc" or "C"
+	if b.disk && !strings.HasSuffix(b.size, "cc") && !strings.HasSuffix(b.size, "C") {
```

Comment:
This was working fine in the past as the disk option was silently ignored in Materialize for cc and C clusters. However this no longer seems to be the case so we need to explicitly skip it in the query builder.